From 44125612ec93f128d83dffed321502e9f5af0f39 Mon Sep 17 00:00:00 2001
From: Craig Kewley <craigx.kewley@intel.com>
Date: Wed, 17 May 2017 15:41:51 +0100
Subject: [PATCH 062/100] ASoC: TDF8532: remove ultrafast mute config

It's not clear what the effects of using ultrafast mode over the default
are. Tests showed no discernible difference so remove it.

Signed-off-by: Craig Kewley <craigx.kewley@intel.com>
---
 sound/soc/codecs/tdf8532.c | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/sound/soc/codecs/tdf8532.c b/sound/soc/codecs/tdf8532.c
index 04ab478..ee0bc48 100644
--- a/sound/soc/codecs/tdf8532.c
+++ b/sound/soc/codecs/tdf8532.c
@@ -138,15 +138,6 @@ static int tdf8532_mute(struct snd_soc_dai *dai, int mute)
 	return tdf8532_cmd_send(tdf8532, data, ARRAY_SIZE(data));
 }
 
-static int tdf8532_set_fast_mute(struct tdf8532_priv *tdf8532)
-{
-	unsigned char data[] = {
-		0x02, (tdf8532->packet_id)++, 0x06,
-			0x80, 0x18, 0x03, 0x01, 0x02, 0x00};
-
-	return tdf8532_cmd_send(tdf8532, data, ARRAY_SIZE(data));
-}
-
 static const struct snd_soc_dai_ops tdf8532_dai_ops = {
 	.shutdown = tdf8532_dai_shutdown,
 	.trigger  = tdf8532_dai_trigger,
@@ -190,11 +181,6 @@ static int tdf8532_i2c_probe(struct i2c_client *i2c,
 
 	i2c_set_clientdata(i2c, tdf8532);
 
-	ret = tdf8532_set_fast_mute(tdf8532);
-
-	if (ret < 0)
-		dev_err(&i2c->dev, "Failed to set fast mute option: %d\n", ret);
-
 	ret = snd_soc_register_codec(&i2c->dev, &soc_codec_tdf8532,
 			tdf8532_dai, ARRAY_SIZE(tdf8532_dai));
 	if (ret != 0)
-- 
1.9.1

